libraries/consul_config.rb

```diff
@@ -76,6 +76,7 @@ class ConsulConfig < Chef::Resource
       attribute(:performance, kind_of: [Hash, Mash])
       attribute(:ports, kind_of: [Hash, Mash])
       attribute(:protocol, kind_of: String)
+      attribute(:raft_protocol, kind_of: Integer, default: false)
```

Comment:
I think we should not specify `default: false` here because Consul can handle the default value by itself.